CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 Version 0.2.0-dev (Changes since version 0.2.0 go here)
 -------------------------------------------------------
 
+* Users can now update and extend sample templates by using the "Update sample templates" button, rather than having to do 2 steps with the previous design.
+* Users can now update and extend prep templates, using the "Update prep template" button in the prep template tab.
+* The 'extend' functionality has been moved from the sample template to the base metadata template object, so now both sample and prep templates can be extended.
+
```

Comment:
It seems like the three points in this section could be synthesized in something like:

* Users can now change values and add columns to sample and prep templates using the <kbd>Update</kbd> button (see the prep template and sample template tabs).

Or something along those lines ...